roadmap.md

```diff
@@ -0,0 +1,58 @@
+(roadmap)=
+# Roadmap
+
+Last updated: February 2022
+
+## Our philosophy
+As stated in our homepage:
+
+> ArviZ is an open source project aiming to provide tools for Exploratory Analysis of Bayesian Models that do not depend on the inference library used.
+
+In this page we describe our current roadmap and specific actions to try and work towards this goal.
+
+## Project roadmap
+We believe both ArviZ and Exploratory Analysis of Bayesian Models are currently relatively established concepts.
+Therefore, generally speaking, as a project we are working on changing ArviZ paradigm/design
+from single monolithical functions that generate complete analysis and visualizations towards
+being more modular.
+
+At the same time, we continue to keep growing our team and publishing documentation as tasks at the same priority level as development.
```

Comment:
```suggestion
At the same time, we continue to keep growing our team and prioritizing publishing documentation at the same level as development.
```

Reply:
I see the original wording is bad, but the proposed changes seems to indicate only publishing docs is at the same level as development, not both publishing docs and growing (maybe also add or change for nurturing) the team

Reply:
Yeah, that's right, and even my suggestion is not great. What about:
```suggestion
At the same time, we are continuing to grow our team and publish documentation at the same priority level as development.
```

Reply:
Ah, actually, I like Colin's suggestion better.